Code/GraphMol/RGroupDecomposition/RGroupGa.h

```diff
@@ -103,7 +102,7 @@ struct GaResult {
 
 class RDKIT_RGROUPDECOMPOSITION_EXPORT RGroupGa : public GaBase {
  public:
-  RGroupGa(const RGroupDecompData& rGroupData,
+  RGroupGa(RGroupDecompData& rGroupData,
```

Comment:
I believe we could keep this const by making the RGroupScorer mutable.  Since the scorer is just caching and not changing results, this its kosher and more explicit in what it is doing I think.

Reply:
(I haven't double checked this, so I could be wrong). But I like int better than changing the API.